### ISSUE TITLE
Don't display the diff when `actual` is `nil`

### ISSUE THREAD

**Issue by positron** [pjstadig/humane-test-output#25]
Feature Request. Feel free to close if you don't like the idea.

I get output like this a lot when developing:

```clojure
expected: {:some
           {:medium-sized-clojure :data-structure
            :that-takes :a}
           :few :lines}
actual: nil
diff: - (... that same data structure repeated ....)
```

I don't think it adds any value to display the diff when one of the arguments is `nil`.

**Comment by pjstadig** [comment]
I do like this, and would like to see it implemented.

**Comment by pjstadig** [comment]
This has been released with 0.11.0
